app/src/main/java/com/example/triptracker/view/profile/UserProfileEditScreen.kt

```diff
@@ -252,7 +252,9 @@ fun UserProfileEditScreen(
                                       }
                                     },
                                     modifier =
-                                        Modifier.height(65.dp).padding(bottom = 5.dp, end = 30.dp),
+                                        Modifier.height(65.dp)
```

Comment:
What arer these changes ? Should have mentioned them in the PR description, I assume they are UI changes

Reply:
Just ktfmt doing it's thing, UI stays the same